index.js

```diff
@@ -25,11 +25,12 @@ Toolkit.run(async tools => {
     return
   }
 
+  const majorWords = process.env['INPUT_MAJOR-WORDING'].split(',')
```

Comment:
`process.env['INPUT_MAJOR-WORDING'] ? process.env['INPUT_MAJOR-WORDING'].split(',') : ['BREAKING CHANGE', 'major']`

Reply:
I have added these values as default in the `action.yml`

Should I move them?

Reply:
ahh ok. than its fine